docs/ios/open-source/manual-instrumentation/custom-attributes.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Custom Attributes
+description: Add contextual data to your app's sessions and spans
+sidebar_position: 3
+---
+
+# Custom Attributes
```

Comment:
Is this what it's called in dashboard? Is it attributes or properties? If the naming convention is different between SDK and dash, note that.